formula/src/main/java/io/reactivex/rxjava3/disposables/FormulaDisposableHelper.kt

```diff
@@ -5,7 +5,7 @@ import java.util.Objects
 /**
  * Used due to issue with [Disposable.fromRunnable], please ignore outside of the internal library.
  */
-object FormulaDisposableHelper {
+internal object FormulaDisposableHelper {
```

Comment:
Oh weird... I thought this would fail since it is a different package from where it is getting called...